docs/source/models/intervention_models/alzheimers/hypothetical_treatment.rst

```diff
@@ -30,4 +30,232 @@
+from pre-clinical to MCI state. This progression rate is set by a :ref:`time-dependent hazard function <2021_cause_alzheimers_presymptomatic_mci_transition_data_table>`
+which is multiplied by a hazard ratio H < 1 when a simulant has an active treatment effect in order to slow the progression.
+This effect can wane over time (udpated each time step) and when the effect fully expires, H returns to 1. 
+
+This treatment is hypothetical and we don't have confirmed information about the mechanism.
+
+
+.. list-table:: Affected Outcomes
+  :widths: 15 15 15 15
+  :header-rows: 1
+
+  * - Outcome
+    - Effect
+    - Modeled?
+    - Note (ex: is this relationship direct or mediated?)
+  * - Hypothetical treatment effect
+    - Slows progression to MCI (time-dependent effect size)
+    - Yes
+    - 
+
+
+
+Vivarium Modeling Strategy
+--------------------------
+
+.. graphviz::
+
+    digraph treatment_states {
+        rankdir = TB;
+        el [label="BBBM test eligible [variable]"]
+        test [label="BBBM test received", style=dashed, shape=box]
+
+        pos [label="BBBM test positive", style=dashed, shape=box]
+        neg [label="BBBM test negative [3 y]"]
+
+        wait [label="In treatment/ Waiting for treatment [6 mo]"]
+        no_treat [label="No treatment effect [permanent]"]
+
+        treat [label="Full treatment effect LONG [5 y]"]
+        wane [label="Waning treatment effect LONG (update each step) [9 y]"]
+
+        treat_short [label = "Full treatment effect SHORT [6 mo]"]
+        wane_short [label = "Waning treatment effect SHORT (update each step) [2.5 y]"]
+
+
+        el -> test [label = "tested"]
+        el -> el [label = "not tested"]
+
+        test -> pos  [label = "(90%)", style=dashed]
+        test -> neg [label = "(10%)", style=dashed]
+        neg -> el [label = "test re-eligible"]
+
+        pos -> wait [label = "decides to initiate treatment (I)", style=dashed]
+        pos -> no_treat [label = "decides not to initiate treatment (1 - I)", style=dashed]
+
+        wait -> treat [label = "completes (90%)"]
+        wait -> treat_short [label = "discontinues (10%)"]
+
+        treat_short -> wane_short
+        wane_short -> no_treat
+
+        treat -> wane 
+
+        wane -> no_treat
+    }
+
+The diagram above illustrates how a simulant should progress through the various testing and treatment related 
+states defined by the client. Each simulant may transition to a new state on each time step. 
+
+Most states have a fixed duration (a multiple of the 
+time step length) where simulants will transition after :math:`\text{duration} / \text{time step}` time steps. 
+The duration is marked in the state node in brackets eg [6 mo]. As desribed in the :ref:`testing intervention <alzheimers_testing_intervention_bbbm>`, 
+some simulants in the `BBBM test eligible` state may transition to tested immediately (low propensity value), some may always self-transition
```

Comment:
by "self-transition" you just mean don't get tested? (i.e. eligible -> eligible)